corehq/pillows/domain.py

```diff
@@ -1,42 +1,22 @@
-import copy
-from corehq.apps.accounting.models import Subscription
 from corehq.apps.change_feed.consumer.feed import KafkaChangeFeed, KafkaCheckpointEventHandler
 from corehq.apps.change_feed import topics
-from corehq.apps.domain.dbaccessors import count_downloads_for_all_snapshots
 from corehq.apps.domain.models import Domain
 from corehq.elastic import get_es_new
 from corehq.pillows.mappings.domain_mapping import DOMAIN_INDEX_INFO
-from django_countries.data import COUNTRIES
 from pillowtop.checkpoints.manager import get_checkpoint_for_elasticsearch_pillow
 from pillowtop.pillow.interface import ConstructedPillow
 from pillowtop.processors import ElasticProcessor
 from pillowtop.reindexer.change_providers.couch import CouchViewChangeProvider
 from pillowtop.reindexer.reindexer import ElasticPillowReindexer, ReindexerFactory
 
 
-def transform_domain_for_elasticsearch(doc_dict):
-    doc_ret = copy.deepcopy(doc_dict)
-    sub = Subscription.visible_objects.filter(subscriber__domain=doc_dict['name'], is_active=True)
-    doc_ret['deployment'] = doc_ret.get('deployment', None) or {}
-    countries = doc_ret['deployment'].get('countries', [])
-    doc_ret['deployment']['countries'] = []
-    if sub:
-        doc_ret['subscription'] = sub[0].plan_version.plan.edition
-    for country in countries:
-        doc_ret['deployment']['countries'].append(COUNTRIES[country].upper())
-    if doc_dict.get('copy_history'):
-        doc_ret['full_downloads'] = count_downloads_for_all_snapshots(doc_dict['copy_history'][-1])
-    return doc_ret
```

Comment:
Its not clear to me why this was deleted. should just the 

```
    if doc_dict.get('copy_history'):	
        doc_ret['full_downloads'] = count_downloads_for_all_snapshots(doc_dict['copy_history'][-1])	
```

be deleted?

Reply:
c0e27a42cd5

When I did this I thought "deployments" were going to be removed altogether, but I'm not longer sure those are exchange-only. Probably not going to dig into that question in this PR.

...also definitely didn't mean to delete the subscription.